index.html

```diff
@@ -68,6 +68,182 @@ <h2>I'am a Software Developer</h2>
         </div>
         <div class="background"></div>
       </section>
+      <section class="works">
```

Comment:
[OPTIONAL] Please kindly make sure you set the `padding-bottom` property accordingly so you have some space between the `works` section and the `about-me` section. Please refer to the screenshot below: 

![Screenshot from 2022-02-12 02-08-02](https://user-images.githubusercontent.com/67608876/153670372-7487d144-cb94-4ef5-858a-2c573c259a66.png)